### ISSUE TITLE
Error with torchvision.models.vgg16_bn

### ISSUE THREAD

**Issue by yxchng** [pytorch/vision#212]
The following is the error I face when using vgg16_bn:

<img width="926" alt="screen shot 2017-07-27 at 7 32 40 pm" src="https://user-images.githubusercontent.com/10518587/28668469-85f18e28-7302-11e7-896f-17d6fffca366.png">

Can I know if I am using it wrongly or is it a bug? Thanks

**Comment by kovacspeter** [comment]
The problem is that you have old version of pytorch where there are no pretrained VGG batch normalized models. We need someone to put newest version on pip. 

Hope that someone will notice this thread because I would also appreciate it. 

Is there any way I can help to do it?

**Comment by alykhantejani** [comment]
Hi, is this still an issue? The following works for me:
```Python
from torchvision.models import vgg16_bn
mod = vgg16_bn(pretrained=False)
```
Using torchvision from master and pytorch `0.2.0+faef95b` (0.2 should be fine though)

**Comment by fmassa** [comment]
We need to cut a new release of `torchvision` soon, the version in master has already a fix for this

**Comment by fmassa** [comment]
As this works in master and we will be cutting a release soon, I'm closing this issue.
